azext_iot/_help.py

```diff
@@ -1352,9 +1360,13 @@
 ] = """
     type: command
     short-summary: Create an enrollment group in an Azure IoT Hub Device Provisioning Service.
+    long-summary: |
+                  Please provide certificate format using Base64 ASCII encoding and the certificate
+                  should have matching BEGIN and END segments, for example:
+                  start with '-----BEGIN CERTIFICATE-----' and end with '-----END CERTIFICATE-----''.
     examples:
     - name: Create an enrollment group '{enrollment_id}' in the Azure IoT provisioning service
-            '{dps_name}' in the resource group '{resource_group_name} using an intermediate certificate as primary certificate'.
+            '{dps_name}' in the resource group '{resource_group_name} using an intermediate certificate as primary certificate.
```

Comment:
nit, some of the example names/descriptions end in period like this one, and others the period was removed, this comment is to strive for consistency